definitions/divorce/json/FixedLists/FixedLists.json

```diff
@@ -561,7 +561,8 @@
     "LiveFrom": "01/01/2017",
     "ID": "courtEnum",
     "ListElementCode": "buryStEdmunds",
-    "ListElement": "Bury St Edmunds"
+    "ListElement": "Bury St Edmunds",
+    "DisplayOrder": 3
```

Comment:
You've added display order 3 here for the ID "courtEnum" and then display orders 1-2 and 4-10 for the ID "DnCourtEnum".

Reply:
Just curious if this was meant to be done for "DnCourtEnum"